### DISCUSSION TITLE
【API翻译功能改进】增加电子书多页翻译和关联上下文字幕翻译设置

### DISCUSSION THREAD

**Discussion by LaLuna302** [immersive-translate/immersive-translate#721] [Ideas]
无论是 OpenAI 还是 DeepL 都能实现关联上下文的智能翻译，一次翻译的字符量越多，翻译质量就越高。若是逐行翻译或段落翻译，关联上下问的优势将无法体现。增加电子书整页或多页同时翻译和关联上下文字幕翻译的设置选项，可解决因逐行翻译或段落翻译导致速度和准确率下降的问题。

**Comment by Andy-AO** [comment]
如果引入Claude 100K就好了，那样可以翻译很长的上下文，准确性估计会显著提高。
